.github/workflows/main.yml

```diff
@@ -55,7 +55,7 @@ jobs:
       fail-fast: true
       matrix:
         os: [ubuntu-latest, windows-latest]
-        python-version: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
dropped support for py3.9 because it's old and we would like to use new type annotation syntax